### DISCUSSION TITLE
Plugin: No language version annotation found :(

### DISCUSSION THREAD

**Discussion by tastytea** [misskey-dev/misskey#9719] [Q&A] [answered]
Hi, i'm trying to write a plugin (for Misskey 13.1.8) and started by copy-pasting simple examples into `/settings/plugin/install`. However, i always get the error message “Plugin: No language version annotation found :(”

Looking at [the code](https://github.com/misskey-dev/misskey/blob/ae63a1f494900c3e4a0d29047a0fe42daeacc82a/packages/frontend/src/pages/settings/plugin.install.vue#L45), it seems i have to write a string starting with `0.12.` somewhere, but i can't figure out where and i can't find it in [the docs](https://misskey-hub.net/docs/advanced/create-plugin.html).

**Comment by Weathercold** [comment, marked as answer]
getLangVersion is defined here https://github.com/syuilo/aiscript/blob/8ab9c279009fe8453c9a65cf3bb7b359d081bb56/src/interpreter/util.ts#L152
First line of the script needs to be `/// @0.12.*`

**Comment by tastytea** [comment]
> getLangVersion is defined here https://github.com/syuilo/aiscript/blob/8ab9c279009fe8453c9a65cf3bb7b359d081bb56/src/interpreter/util.ts#L152 First line of the script needs to be `/// @0.12.*`

* with `/// @0.12.*` it returns “No language version annotation found :(”
* with `/// @0.12.0` or `/// @0.13.2` it returns “Syntax error :(”
* with `/// @0.12` or `/// @0.13` it returns “aiscript version '0.12' is not supported :(”

this is my code:

```aiscript
/// @0.12.*
### {
  id: "5f9f1949-f245-4229-bfeb-6e01a53df984"
  name: "Example Plugin"
  version: "0.0.0"
  author: "syuilo"
}

@main() {
  Mk:register_post_form_action("fujiwara" @(form update) {
    #text = `{Arr:join(Str:split(form.text), "゛")}゛`
    update("text" text)
  })
}

main()
```

**Reply by Weathercold** [reply]
Try `let text =` instead of `#text =`

**Reply by tastytea** [reply]
ah, thanks! that alone didn't help but when emptying the main function completely it did work with `/// @0.13.2`.
